CHANGES.rst

```diff
@@ -179,6 +179,9 @@ New features:
 
 Bug fixes:
 
+- Fix ft._updateProperty so it doesn't break when receiving an empty value.
+  This happens when an DX FTI is part of a Generic Setup baseline import.
+  [jaroel]
```

Comment:
You need to put this log entry in a Towncrier conform news file.

Reply:
Thanks! I've moved the entry to `news/85.bugfix`